ui/app/components/mfa/mfa-login-enforcement-form.js

```diff
@@ -94,6 +97,11 @@ export default class MfaLoginEnforcementForm extends Component {
     this.authMethods = mounts.map((auth) => auth.type);
```

Comment:
Probably outside of scope - but when testing MFA last week i notice this doesn't filter out duplicates. if you wanted to throw a `filter()` here
<img width="967" alt="Screenshot 2024-10-08 at 9 35 49 AM" src="https://github.com/user-attachments/assets/e39cde8a-bd15-4a49-96ba-38bfb4622ed6">

Reply:
oh good catch. i'll add that in.